cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -164,6 +165,7 @@ func (o *Options) complete() error {
 	shootquotavalidator.Register(o.Recommended.Admission.Plugins)
 	shootdns.Register(o.Recommended.Admission.Plugins)
 	shootvalidator.Register(o.Recommended.Admission.Plugins)
+	seedvalidator.Register(o.Recommended.Admission.Plugins)
```

Comment:
Please add docs to https://github.com/gardener/gardener/blob/master/docs/concepts/apiserver_admission_plugins.md